api_app/models/schemas/airlock_request.py

```diff
@@ -29,15 +29,33 @@ class Config:
         }
 
 
-class AirlockRequestInList(BaseModel):
-    airlockRequests: List[AirlockRequest] = Field([], title="Airlock Requests")
+class AirlockRequestWithAllowedUserActions(BaseModel):
+    airlockRequest: AirlockRequest = Field([], title="Airlock Request")
+    allowed_user_actions: List[str] = Field([], title="actions that the requesting user can do on the request")
+
+    class Config:
+        schema_extra = {
+            "example": {
+                "allowed_user_actions": [AirlockActions.Review]
+            }
+        }
+
+
+class AirlockRequestWithAllowedUserActionsInList(BaseModel):
+    airlockRequests: List[AirlockRequestWithAllowedUserActions] = Field([], title="Airlock Requests")
 
     class Config:
         schema_extra = {
             "example": {
                 "airlock_requests": [
-                    get_sample_airlock_request("933ad738-7265-4b5f-9eae-a1a62928772e", "121e921f-a4aa-44b3-90a9-e8da030495ef"),
-                    get_sample_airlock_request("123ad738-1234-4b5f-9eae-a1a62928772e", "457e921f-a4aa-44b3-90a9-e8da030412ac"),
+                    {
+                        "airlockRequest": get_sample_airlock_request("933ad738-7265-4b5f-9eae-a1a62928772e", "121e921f-a4aa-44b3-90a9-e8da030495ef"),
```

Comment:
I don't think it follows the convention... can you combine in one func?

Reply:
done